checkstyle/checkstyle.xml

```diff
@@ -60,10 +60,9 @@
     <module name="TypeName"/>
     
     <!-- whitespace -->
-    <module name="GenericWhitespace"/>
```

Comment:
Seems like a useful check to have. Probably better to locally suppress it in the few locations where it's broken.